tests/playwright_test.go

```diff
@@ -1,32 +1,30 @@
 package playwright_test
 
 import (
-	"log"
+	"path/filepath"
+	"testing"
 
 	"github.com/mxschmitt/playwright-go"
+	"github.com/stretchr/testify/require"
 )
 
-func assertErrorToNil(message string, err error) {
```

Comment:
The reason why I was doing it like this was that it gets shown as an example in the docs.

Refere: https://blog.golang.org/examples

But for some reason it does not get shown anymore: https://pkg.go.dev/github.com/mxschmitt/playwright-go@master#Run

Reply:
We do have dedicated examples, so there isn't any need for it IMO